docs/docs/understanding/workflows/state.md

```diff
@@ -0,0 +1,66 @@
+# Maintaining state
```

Comment:
We should mention somewhere that state/context is maintained between each .run() call. So, the state is attached to the specific instance of that workflow

Reply:
TIL!